examples/rvc-app/rvc-common/pics/rvc-app-pics-values

```diff
@@ -1,12 +1,14 @@
+PICS_SDK_CI_ONLY=1
+
 RVCCLEANM.S=1
 RVCCLEANM.S.A0000=1
 RVCCLEANM.S.A0001=1
-RVCCLEANM.S.A0002=0
-RVCCLEANM.S.A0003=0
-RVCCLEANM.S.C01.Tx=1
 RVCCLEANM.S.C00.Rsp=1
+RVCCLEANM.S.C01.Tx=1
 RVCCLEANM.S.F00=0
 RVCCLEANM.S.M.CAN_TEST_MODE_FAILURE=1
+PIXIT.RVCCLEANM.MODE_CHANGE_FAIL=1
```

Comment:
These two will be removed, right?

Reply:
They have been removed in [this PR](https://github.com/project-chip/connectedhomeip/pull/31827).